cloudzero/manifest.json

```diff
@@ -0,0 +1,42 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "6f1a61c2-e875-42f7-b8ba-94b191786846",
+  "app_id": "cloudzero",
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "View and analyze your Datadog costs on the CloudZero platform.",
+    "title": "CloudZero",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Cloud costs - Grouped by Provider",
+        "image_url": "images/CloudZero1.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Datadog Costs - Grouped by Service",
+        "image_url": "images/CloudZero2.png"
+      }
+    ],
+    "classifier_tags": [
+      "Supported OS::Linux",
+      "Supported OS::macOS",
+      "Supported OS::Windows",
+      "Offering::Integration",
+      "Category::Cost Management",
+      "Category::Cloud",
+      "Queried Data Type::Usage"
```

Comment:
The submitted/queried data types are set values, and only currently support the following: Metrics, Traces, Logs, Events, Incidents. Since none of those are relevant for your integration, you can remove this